src/Microsoft.DotNet.Wpf/src/Shared/System/Windows/Markup/ReflectionHelper.cs

```diff
@@ -193,11 +178,6 @@ internal static Type GetFrameworkType(string assemblyName, Type type)
             return type;
         }
 
-        internal static Type GetMscorlibType(Type type)
```

Comment:
can you add this back? it looks like this is still in use by other WPF binaries

Reply:
not that you could know at this point, we'll get there!

Reply:
No probs, restored with an `#if SYSTEM_XAML` check